test/unittests/state_transition.cpp

```diff
@@ -211,6 +212,9 @@ void state_transition::export_state_test(
     jtx["gasLimit"][0] = hex0x(tx.gas_limit);
     jtx["value"][0] = hex0x(tx.value);
 
+    if (tx.type >= Transaction::Type::access_list)
```

Comment:
This seems hacky (but I also remember the JSON representation of access list was a mess). Can you add a comment what it does? Can we actually dump the access list when it is not empty?

Reply:
> Can we actually dump the access list when it is not empty?

not sure what you mean here?

Reply:
Oh I see now. The non-empty `"accessLists"` is handled next. Just add a comment: "force access list output even if empty".